apps/web/app/room/[slug]/components/chat.tsx

```diff
@@ -41,7 +41,7 @@ export const Chat: React.FC<Chat> = ({ messages, onSend }) => {
 
   return (
     <div className="align-center flex h-full w-full flex-col items-center justify-center gap-2">
-      <ScrollArea className="flex w-full flex-1 items-center gap-2 rounded-lg border p-4">
+      <ScrollArea className="md:min-h-5/6 flex max-h-[400px] w-full flex-1 items-center gap-2 rounded-lg border p-4 md:max-h-[84vh]">
```

Comment:
`md:max-h-[84vh]`


This week I was thinking about it, but I didn't have sure if this is the right approach here, because will be easy to break this layout with small changes on other components, what do you think?

Reply:
Yes, for sure; although the mobile version is okay for now, it is not a permanent fix for the desktop; it is only something I did to prevent the infinite height. I guess users will only use the chat for short conversations, so I didn't think much in-depth about it. This fix works for 1080p, but we can think of a number of solutions to solve it completely. 

One of them is making the chat component sticky on the desktop. That way, we would be able to control it more independently.

Another is using the chat to define the page's height; the video container would use it as a reference.

But we would probably have to refactor more of the page's structure to solve this completely.

I can take a closer look at it if you want in this PR or another. Let me know your preference.

Reply:
I believe it can be done in a future pr, this is already a great improvement for now. Thak you for explanations